mpi4jax/_src/jax_compat.py

```diff
@@ -94,12 +80,3 @@ def check_jax_version():
             f"    $ pip install -U mpi4jax\n\n"
             f"You can set the environment variable `{warn_envvar}=1` to silence this warning."
         )
-
-    # check version of jaxlib
-    import jaxlib
-
-    if versiontuple(jaxlib.__version__) < versiontuple(JAXLIB_MINIMUM_VERSION):
```

Comment:
Should we still leave an explicit version check? With pip is relatively easy/commmon to end up with incompatible dependency versions.
But that's also on the user...

Reply:
Agree that it's easy, but I'd rather stick to Python library convention *not* to enforce dependencies at runtime. 6 month deprecation cycle should be wide enough to avoid most conflicts.